interfaces/Output_from_core.atd

```diff
@@ -106,6 +107,13 @@ type metavar_value
   unique_id: unique_id;
 }
 
+type svalue_value
+  <python decorator="dataclass(frozen=True)"> = {
+  svalue_start: position;
```

Comment:
I think these two positions should be optional. We don't try very hard to assign positions to propagated constants so I guess this will filter out many cases.